ros/src/twist_controller/dbw_node_perf.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import Bool
+from dbw_mkz_msgs.msg import ThrottleCmd, SteeringCmd, BrakeCmd, SteeringReport
+from geometry_msgs.msg import TwistStamped, PoseStamped
+from styx_msgs.msg import Lane, Waypoint
+from scipy.spatial import KDTree
+import math
+
+class DBWNodePerf(object):
+    def __init__(self):
+        rospy.init_node('dbw_node_perf')
+
+        self.freq = 2 # Hz
```

Comment:
Is this the frequency we want to use ?

Reply:
This is a good question. 
Honestly, I didn't really think when chose the value.

The car drives 40MPH = 11.1(1) meters per second.
The node will calculate MSE every 0.5 seconds, or ~5 meters.
The distance between the waypoints is much less than the 5 meters.
So looks like it makes sense to increase is.

Let me change it to `40` and rerun tests.

I don't think the final result will change, but we will get more accurate numbers which are good for the write-up.

Reply:
Updated.